packages/dev-env/src/seed/basic.ts

```diff
@@ -155,6 +155,8 @@ export default async (
     })
   }
 
+  await sc.network.processAll()
+
```

Comment:
All the other seeds let the caller perform the final `processAll()`— does it cause any issues if we stick with that for now?

Reply:
I see in the new test suite it works the same way:
```ts
    await basicSeed(sc)
    await network.processAll()
```

Reply:
nope, don't think there's any issue with that. just cleaned up!